### ISSUE TITLE
Sections not collapsed on mdwiki

### ISSUE THREAD

**Issue by kelson42** [openzim/mwoffliner#1860]
on small screens

**Comment by VadimKovalenkoSNF** [comment]
I'm not sure it is about small screens... Could you give me a screenshot with the example? If you tested mwoffliner with disabled MCS, sections are not collapsed there at all as I described in https://github.com/openzim/mwoffliner/issues/1866

**Comment by kelson42** [comment]
In MWoffliner there is a piece of code wich collapse all sections on "small" screens based on the screen wodth.

See https://library.kiwix.org/viewer#mdwiki_en_all_2023-09/A/Spina_bifida

**Comment by VadimKovalenkoSNF** [comment]
I've checked the [code piece](https://github.com/openzim/mwoffliner/blob/main/res/script.js#L16) (in fact, client script) responsible for this functionality. It requires `details` and `sections` tags in the output which are not supported for now.

**Comment by farfromrefug** [comment]
Not sure if it is related. But on my android phone if I open  a Wikipedia artitle al sections are "open" by default. It is disturbing and not very practical (web is all sections closed). Could it be related?

**Comment by Jaifroid** [comment]
> Not sure if it is related. But on my android phone if I open a Wikipedia artitle al sections are "open" by default. It is disturbing and not very practical (web is all sections closed). Could it be related?

Yes, this is happening for all recent Wikipedia ZIMs, and must be due to the removal of summary/detail tags as outlined here and in #1915.

**Comment by kelson42** [comment]
The modern end-points don't collapse sections anymore. Closing.
